easybuild/easyblocks/generic/conda.py

```diff
@@ -57,11 +59,19 @@ def extract_step(self):
             super(Conda, self).extract_step()
 
     def install_step(self):
-        """Install software using 'conda env create' or 'conda create' & 'conda install'."""
+        if get_software_root('anaconda') or get_software_root('miniconda'):
```

Comment:
```suggestion
        if get_software_root('anaconda2') or get_software_root('miniconda2') or get_software_root('anaconda3') or get_software_root('miniconda3'):
```
The current batch of easyconfigs that match this dependency are called `Anaconda2/Anaconda3` and `Miniconda2/Miniconda3`.

Note that Mamba also includes a `conda` command so we do actually need to check the dependency to make the right choice.